desktop-wm/cwm/autobuild/overrides/usr/share/xsessions/cwm.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=CWM
+Comment=A clean and simple window manager for X inspired by evilwm
```

Comment:
Why is this inconsistent with the package description? Also, in this case, I would take the "Comment" field to describe what CWM stands for.